### PR TITLE
group_vars: add con host group configuration

### PR DESCRIPTION
**Pull request by xresende**
- Add `con`host group
- Add `con` and `elp`host groups to control-room Desktop deploys
- Circumvent deprecated repository signature from NSLS-II
- Circumvent "broken" `phoebus` download services.

### CODE REVIEW

File: roles/lnls-ans-role-cs-studio/vars/main.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ csstudio_diirt_preferences:
 
 csstudio_phoebus_integration_preferences:
   - regexp: "^org.csstudio.phoebus.integration/phoebus_home="
-    line: "org.csstudio.phoebus.integration/phoebus_home={{ phoebus_directory }}"
+    line: "org.csstudio.phoebus.integration/phoebus_home=/opt/phoebus-{{ pkg_version_phoebus }}"
```
Comment (xresende) [inline roles/lnls-ans-role-cs-studio/vars/main.yml:56]: this variable is undefined when `phoebus` role is commented out.

File: roles/lnls-ans-role-desktop-apps/tasks/setup-Debian.yml [hunk 2]
```diff
@@ -15,6 +15,8 @@
     state: "{{ desktop_packages_state }}"
     install_recommends: "{{ desktop_install_recommends }}"
     default_release: "{{ desktop_default_release | default(omit) }}"
+    allow_unauthenticated: true
+    force: true
```
Comment (xresende) [inline roles/lnls-ans-role-desktop-apps/tasks/setup-Debian.yml:19]: not sure how dangerous this is. I have to use it do down downgrade `ca-certificates` repos in con newly created debian-9.12 host.

Comment (xresende) [reply roles/lnls-ans-role-desktop-apps/tasks/setup-Debian.yml:19]: @lerwys  pointed out frightening warnings on the use of this `force: true` option. I am removing this option...

File: roles/lnls-ans-role-repositories/tasks/setup-Debian.yml [hunk 3]
```diff
@@ -6,6 +6,8 @@
       - apt-transport-https
     state: present
     update_cache: true
+    allow_unauthenticated: true
+    force: true
```
[the remark this answers is not in the document]
Comment (xresende) [reply roles/lnls-ans-role-repositories/tasks/setup-Debian.yml]: it is outdated. I deleted it in commit  081686d

**Comment by xresende** [comment]
> Sorry for the delay. Good news. We don't need to allow unauthenticated packages anymore. NSLS-II has updated their keys. Now it expires on: `2022-07-07`
> 
> I think we can remove all ` allow_unauthenticated: true` now

I'll remove this flag and try running.

File: roles/lnls-ans-role-repositories/tasks/setup-Debian.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (lerwys) [reply roles/lnls-ans-role-repositories/tasks/setup-Debian.yml]: Ok. I see no problem in removing it.

**Pull request merged**